steps/monitoring-apps-permissions/1.0.0/step.yml

```diff
@@ -0,0 +1,117 @@
+title: Mobile apps permissions monitoring
+summary: |
+  Check and monitor mobile applications permissions
+description: |
+  Check mobile applications permissions and compare with configured values to be warned when new permissions have been added
+
+  ### Usage
+
+  To give to our step the informations about the expected values, you have:
+  - create a config file (which should be added to your project repository) and set `config_file_path`  to find them. You can find a config example file [here](https://github.com/imranMnts/bitrise-step-monitoring-apps-permissions)
+  - **OR** set these keys with expected values directly on Bitrise side
+    - android_permission_count
+    - ios_permission_count
+
+  ### Troubleshooting
+
+  - Be aware, you have to launch this step after [apps-decompiler](https://github.com/imranMnts/bitrise-step-apps-decompiler) step
+website: https://github.com/odemolliens/bitrise-step-monitoring-apps-permissions
+source_code_url: https://github.com/odemolliens/bitrise-step-monitoring-apps-permissions
+support_url: https://github.com/odemolliens/bitrise-step-monitoring-apps-permissions/issues
+published_at: 2021-01-21T12:42:50.16025+01:00
+source:
+  git: https://github.com/odemolliens/bitrise-step-monitoring-apps-permissions.git
+  commit: 51ba8f1810bb8e204730a330858a7f040bda99cc
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- ios
+- android
+- react-native
+- flutter
+type_tags:
+- utility
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: gnu-sed
+is_requires_admin_user: false
+is_always_run: true
```

Comment:
Do you want the `is_always_run` field to be `true`? 
```
is_always_run: if true the step will be executed even if a previous step failed during the build. 
Default is false.
- if type_tags is 'notification' this needs to be true
```

Reply:
We have to keep it to true because when a quality step (eg. monitoring-apps-sizes step) finds something, it generates an error so it will cancel all other quality checks which will launch after this one.

I switched `is_always_run`to false on `apps-decompiler` step and we are checking at the top of the script if we have or not the decompiled app to not try to do something without that